File: AutomaticBrewingCoffeeFE/interfaces/organization.ts
import { EBaseStatus } from "@/enum/base";

export interface Organization {
    organizationId: string;
    organizationCode?: string;
    name: string;
    description: string;
    contactPhone: string;
    contactEmail: string;
    logoUrl: string;
    taxId: string;
    status: EBaseStatus;
    createdDate: string;
    updatedDate: string | null;
    deletedDate?: string | null;
    isDeleted?: boolean;
}

File: AutomaticBrewingCoffeeBE/AutomaticBrewingCoffee.Main/Domains/Models/Organization.cs
using System.ComponentModel.DataAnnotations;

namespace AutomaticBrewingCoffee.Domain.Models;

public class Organization : BaseModel
{
    [Key] [StringLength(50)] [Required] public string OrganizationId { get; set; } = null!;
    [StringLength(100)] [Required] public string Name { get; set; } = null!;
    [StringLength(100)] public string OrganizationCode { get; set; } = null!;
    [StringLength(450)] public string? Description { get; set; }
    [StringLength(100)] public string? ContactPhone { get; set; }
    [StringLength(100)] public string? ContactEmail { get; set; }

    [StringLength(450)] public string? LogoUrl { get; set; }
    [StringLength(200)] public string? TaxId { get; set; }
    [StringLength(10)] [Required] public string Status { get; set; } = null!;
}